srcpkgs/qemu/template

```diff
@@ -64,22 +65,14 @@ do_configure() {
 		--disable-xen --enable-tpm \
 		--enable-vhost-net --enable-vnc-png --enable-virtfs \
 		--enable-libusb --disable-glusterfs --enable-snappy --enable-usb-redir \
-		--enable-pie --localstatedir=/var --enable-docs \
+		--enable-pie --localstatedir=/var --enable-docs --disable-strip \
```

Comment:
Is --disable-strip really necessary, now with the strip wrapper being in place again?

Reply:
If that's SOP then no. I threw that in because the whole template was put together without consideration of debuginfo so I didn't know what would be missing. I can remove it if it would be cleaner.

Reply:
...and the strip wrapper fires anyway so yeah I'll remove the flag